core/services/chainlink/application.go

```diff
@@ -303,6 +306,28 @@ func NewApplication(opts ApplicationOpts) (Application, error) {
 			chainSet,
 			globalLogger,
 		)
+
+		// TODO [relay]: add flag for this experimental job type?
+		globalLogger.Debug("Off-chain reporting v2-relay enabled")
+		// master/delegate relay is started once, on app start, as root subservice
+		// TODO [relay]: move relayers creation outside OCR2 context (relayers will be multi-protocol)
```

Comment:
We will do that when necessary

Reply:
I think this is a ways away and totally unclear how the relayers will be used outside OCR as they are super OCR specific at the moment. I'd hold off on it